apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.TargetedSessionPoolTests.cs

```diff
@@ -213,6 +214,38 @@ public async Task AcquireAsync_ZeroPoolSizeAcquireUpToMaximum()
                 });
             }
 
+            [Fact]
+            public async Task AcquireAsync_UpToMaxActiveSessionsOneAtATime()
+            {
+                var pool = CreatePool(true);
+                var client = (SessionTestingSpannerClient)pool.Client;
+                await client.Scheduler.RunAsync(async () =>
+                {
+                    for (int i = 0; i < pool.Options.MaximumActiveSessions; i++)
+                    {
+                        await pool.AcquireSessionAsync(new TransactionOptions(), default);
+                    }
+                });
```

Comment:
nit: Maybe assert agains some statistics? And same with the test below.

Reply:
For the sake of git simplicity, I'll add those in a commit in PR #2633. Will do that now though while rebasing :)